packages/subgraph/schema.graphql

```diff
@@ -635,8 +638,26 @@ interface IProposal {
   executionTxHash: Bytes
 }
 
-type TransactionActionsProposal implements IProposal @entity {
+"""
+Temporary entity until Actions have been removed as part of the split
+"""
+type TransactionAction @entity {
```

Comment:
I wonder why we would remove this from the osx. Each action has `to, value, data` that still should be stored on OSX.  What would you have instead ?

Reply:
We will rename back to `Action` when we remove the plugin data. By separating to `TransactionAction` I can keep the Executed Actions on the DAO (`TransactionAction`) separated from the `Action` entity used by plugins, which minimizes changes in the codebase